### ISSUE TITLE
FormControls.Static support for components in value?

### ISSUE THREAD

**Issue by eventhough** [react-bootstrap/react-bootstrap#1023]
I'd like to display a link as the value for FormControls.Static but I get this warning:

```
Warning: Failed propType: Invalid undefined `value` supplied to `Static`. Check the render method of `DetailForm`.
```

If you swap value with children you get a similar warning:

```
Warning: Failed propType: Invalid undefined `children` supplied to `Static`. Check the render method of `DetailForm`.
```

It looks like FormControls.Static currently takes either a value or children and requires that the value be a string.

Is there a way for us to have the option to use a component?

[http://react-bootstrap.github.io/components.html#input-static](http://react-bootstrap.github.io/components.html#input-static)

**Comment by aabenoja** [comment]
At the moment the expected values are numbers and strings. It is, however, very simple to change this to `PropTypes.node` [in this section](https://github.com/react-bootstrap/react-bootstrap/blob/master/src/utils/childrenValueInputValidation.js#L10-L11). A pull request is welcome if you want to hop right on it, though I could get around to it in a few hours.

**Comment by jontewks** [comment]
Ran into this issue as well, so submitted a PR #1269 to fix.

**Comment by taion** [comment]
Fixed by #1269.
